Client/src/Components/TabPanels/ProfileSettings/ProfilePanel.jsx

```diff
@@ -23,63 +24,38 @@ const ProfilePanel = () => {
   const [isLoading, setIsLoading] = useState(false);
 
   //for testing, will tweak when I implement redux slice
+  const idToName = {
+    "edit-first-name": "firstName",
+    "edit-last-name": "lastName",
+    "edit-email": "email",
```

Comment:
Can you double check these mappings? I believe the backend is is expecting all lower case, ie `firstname` and `lastname`.

Reply:
Sure! on that topic should I also change the naming for the password panel form fields? I'm not sure how the backend is currently set up and what fields should correspond to what. 

This is how I have the mappings for the new password form set atm:
`"edit-current-password": "currentPassword"` - current (old) password
`"edit-new-password": "password"` 
`"edit-confirm-password": "confirm"`

Reply:
I can't remember exactly what they are either, they are all in the docs for the respective API endpoints though.

If there isn't an endpoint for whichever request you're working on then you can leave them as is and we can revise if needed when the time comes.

Reply:
I don't believe there is an endpoint for resetting password w/o a recovery token currently so I'll leave it as is.

Reply:
I believe you'll be hitting

POST/api/v1/auth/user/{user ID}

for editing users,so naming conventions should match with that one.

I don't think confirm password field needs to be passed to the backend as match should already be verified on client side.

We can update the backend if we want to verify there as well too though.

Reply:
Makes sense. I'll update the names in that case to match the convention. As for the newpassword and confirm fields I'll leave them as TBD for now and will touch base once I start working on that part.